new-lamassu-admin/src/pages/MachineLogs.js

```diff
@@ -123,16 +123,21 @@ const Logs = () => {
                 <TableHeader className={classes.fillColumn} />
               </TableRow>
             </TableHead>
-            <TableBody>
-              {logsResponse &&
-                logsResponse.machineLogs.map((log, idx) => (
-                  <TableRow key={idx} size="sm">
-                    <TableCell>{formatDate(log.timestamp)}</TableCell>
-                    <TableCell>{log.logLevel}</TableCell>
-                    <TableCell>{log.message}</TableCell>
-                  </TableRow>
-                ))}
-            </TableBody>
+            {logsResponse &&
+              (!logsResponse.machineLogs.length ? (
```

Comment:
I think you can just check for the existence of machineLogs and not necessarily its length.
```suggestion
              (!logsResponse.machineLogs ? (
```

Reply:
You are right, I think I was missing the point of the card. I should check !logsResponse to show the empty message